### PR TITLE
Add App's skeleton

### LINKED ISSUE #4: Complete README

### LINKED ISSUE #5: Add Makefile

### LINKED ISSUE #7: Add project skeleton

### PR DESCRIPTION
**Pull request by fborello-lambda**

### CODE REVIEW

File: Makefile
Comment (JulianGCalderon) [inline Makefile:1]: I'd start with a more atomic Makefile. There are a few things that we don't need yet, we can expand it as we need. Maybe `deps.get` and `phx.server`is enough for now.

**Review by ManuelBilbao — CHANGES_REQUESTED** [review]
Please update the README including the instructions to setup and run the project on dev environment

**Comment by fborello-lambda** [comment]
Makes sense, the Makefile was reduced, and the README has the minimal instructions to setup the project.

**Review by JulianGCalderon — DISMISSED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -7,3 +7,26 @@
...
+- Mix 1.16.1
+- Docker(optional)
+
+# Development Guide
+
+A PostgreSQL database is needed, using Docker and using this setup for the developing stage:
+
+```sh
+docker run \
+        --name lambcast_db \
```
Comment (ManuelBilbao) [inline README.md]:
```suggestion
        --name lambcast_db \
```

There's a trailing space on this line, which breaks the command in two. This cause an error when copy-pasting the command

**Review by ManuelBilbao — APPROVED** [review]

**Review by JulianGCalderon — APPROVED** [review]

**Pull request merged**